winsw.csproj

```diff
@@ -92,6 +92,7 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <None Include="packages.config" />

```

Comment:
NuGet package won't be included to the output if there's no reference in the code.
I can remove the package dependency to keep the repo clian